aptos-move/aptos-release-builder/src/validate.rs

```diff
@@ -0,0 +1,379 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::ReleaseConfig;
+use anyhow::Result;
+use aptos_api_types::U64;
+use aptos_crypto::ed25519::Ed25519PrivateKey;
+use aptos_genesis::keys::PrivateIdentity;
+use aptos_rest_client::Client;
+use aptos_temppath::TempPath;
+use aptos_types::account_address::AccountAddress;
+use futures::executor::block_on;
+use serde::Deserialize;
+use std::{
+    fs,
+    path::{Path, PathBuf},
+    process::Stdio,
+    thread::sleep,
+    time::Duration,
+};
+use url::Url;
+
+#[derive(Clone, Debug)]
+pub struct NetworkConfig {
+    endpoint: Url,
```

Comment:
As discussed offline, let's pass this endpoint into the underlying CLI calls (create proposal, vote, resolve, etc.) so we wouldn't need to point the default aptos cli profile at local testnet.